analysed-packages/dosfstools/version-4.2/dosfstools-4.2-SPDX2TV.spdx

```diff
@@ -0,0 +1,3339 @@
+Copyright (C) 1998-2005 Roman Hodek <Roman.Hodek@informatik.uni-erlangen.de>
+Copyright (C) 2008-2014 Daniel Baumann <mail@daniel-baumann.ch>
+Copyright (C) 2015-2016 Andreas Bombe <aeb@debian.org>
+Copyright (C) 2018 Pali Rohár <pali.rohar@gmail.com> </text>
+
+##File
+
+FileName: dosfstools-4.2.tar.gz/dosfstools-4.2.tar/dosfstools-4.2/COPYING
+SPDXID: SPDXRef-item440950
+FileChecksum: SHA1: 8624bcdae55baeef00cd11d5dfcfa60f68710a02
+FileChecksum: SHA256: 8ceb4b9ee5adedde47b31e975c1d90c73ad27b6b165a1dcd80c7c545eb65b903
+FileChecksum: MD5: d32239bcb673463ab874e80d47fae504
+LicenseConcluded: LicenseRef-License-of-GNU-Licenses
+LicenseComments: <text> The file contains the text of the GPL-3.0. The text of the license is licensed under:
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+Thus we concluded License-of-GNU-Licenses as license of this file. </text>
+LicenseInfoInFile: LicenseRef-GPL-3.0-only
+LicenseInfoInFile: LicenseRef-GPL-3.0
+FileCopyrightText: <text> Copyright (C) 2007 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: dosfstools-4.2.tar.gz/dosfstools-4.2.tar/dosfstools-4.2/configure
+SPDXID: SPDXRef-item440896
+FileChecksum: SHA1: c3a7785644a01ec67f74ddb282740df45fbabfcf
+FileChecksum: SHA256: 535c481612b7233bb26309d2daf40c9bb57d519285869b915e6fce4c83c64f37
+FileChecksum: MD5: 0a2bfea898998ce3a2fe0ceecfc26533
+LicenseConcluded: LicenseRef-FSFUL-98b8c57e8dae68bce1ba5bc67de59a13
+LicenseInfoInFile: LicenseRef-FSFUL
+LicenseInfoInFile: LicenseRef-fsf-free
+FileCopyrightText: <text> Copyright (C) 1992-1996, 1998-2012 Free Software Foundation, Inc.
+Copyright (C) 2012 Free Software Foundation, Inc.
+Copyright (C) 2012 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: dosfstools-4.2.tar.gz/dosfstools-4.2.tar/dosfstools-4.2/compile
+SPDXID: SPDXRef-item440912
+FileChecksum: SHA1: c4f01c6c960bb0c07d1a0836e8c2bd5503aee864
+FileChecksum: SHA256: cf686b3cb1f8f8ffc4f0295671f2fce476f6187e98a456dd2b71ec0c1d2a407c
+FileChecksum: MD5: 28e27b0082703426127843fe36c96cc0
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1999-2018 Free Software Foundation, Inc. Written by Tom Tromey <tromey@cygnus.com>. </text>
+
+##File
+
+FileName: dosfstools-4.2.tar.gz/dosfstools-4.2.tar/dosfstools-4.2/doc/README.dosfsck
+SPDXID: SPDXRef-item440908
+FileChecksum: SHA1: 7b162866e9b9af9c953ba8df2bd5915ce348981f
+FileChecksum: SHA256: 2912131e0a9fa82189fa26d12d70dce3cb73fc3c935ed4a82d76c26db062e66f
+FileChecksum: MD5: 625934b496c765f8810238b6368c8875
+LicenseConcluded: LicenseRef-warranty-disclaimer-45f67d830b5b48e8d64e1114612be857
```

Comment:
this is not the license of the file,the file only contains text. The license is NOASSERTION